CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 .. currentmodule:: wtforms
 
 
+Unreleased
```

Comment:
This should be "Version 2.3.2", with "Unreleased" under it. Can fix that when you make the release PR.